docker/Dockerfile

```diff
@@ -82,13 +83,12 @@ RUN rosdep keys --dependency-types=exec --ignore-src --from-paths src \
 FROM base AS autoware-core
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 ARG ROS_DISTRO
-ARG SETUP_ARGS
 ENV CCACHE_DIR="/root/.ccache"
 
 # Set up development environment
 RUN --mount=type=ssh \
   --mount=type=cache,target=/var/cache/apt,sharing=locked \
-  ./setup-dev-env.sh -y --module all ${SETUP_ARGS} --no-cuda-drivers openadkit \
+  ./setup-dev-env.sh -y --module all openadkit \
```

Comment:
@youtalk this command will not run the `autoware.dev_env.cuda` role.
The `autoware.dev_env.cuda` will only run if `module == 'base' and prompt_install_nvidia=='y'`. 
Is it possible that there is a bug here?

Reply:
This PR changed the Dockerfile to install CUDA drivers on `base` stage.

https://github.com/autowarefoundation/autoware/pull/5159/files#diff-f34da55ca08f1a30591d8b0b3e885bcc678537b2a9a4aadea4f190806b374ddcL27-R28

Reply:
@youtalk what is misleading is the argument `--no-cuda-drivers` which implies that there are no CUDA drivers?.